include/vctr/Miscellaneous/StdOstreamOperator.h

```diff
@@ -26,7 +26,7 @@ namespace vctr
 template <is::anyVctrOrExpression V>
 std::ostream& operator<< (std::ostream& s, const V& vec)
 {
-    if (vec.empty())
+    if (vec.size() == 0)
```

Comment:
Just out if interest: Why?

Reply:
since expressions are only required to implement a size member function, so this didn't work when attempting to print an expression result